### PR TITLE
Create alternate-cell-background-color-by-table-group.md

### PR DESCRIPTION
**Pull request by todorarabadzhiev**

### CODE REVIEW

**Review by madsfarner — APPROVED** [review]
Might be worth mentioning that the '%' is a modulus operator, though most developers would most likely be familiar with this. In addition, the mod operator suggested in the Designer is 'Mod', though both operators work

File: knowledge-base/alternate-cell-background-color-by-table-group.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: Alternate cell background color by table group
```
Comment (nelci592) [inline knowledge-base/alternate-cell-background-color-by-table-group.md:2]: All words except prepositions should be with capital letters

File: knowledge-base/alternate-cell-background-color-by-table-group.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+			<td>Progress® Telerik® Reporting</td>
+		</tr>
+	</tbody>
+</table>
+
+
+## Description
+What [Conditional Formatting](../styling-conditional-formatting) may be used to alternate the detail cell background color based on table groups?  
+
+For alternating detail rows in a table, you may use _Conditional Formatting_ based on the following [Expression](../report-expressions) and set different colors for
```
Comment (nelci592) [inline knowledge-base/alternate-cell-background-color-by-table-group.md:31]: it is recommended to use bold instead of italics

**Review by nelci592 — COMMENTED** [review]
It looks good but I think that it would be better if you replace italics with bold.

**Pull request merged**